ciphers/vigenera_cipher_dictionary_hacker.py

```diff
@@ -0,0 +1,43 @@
+import detectEnglish, vigenereCipher, pyperclip
+
+
+def main():
+    ciphertext = "Tzx isnz eccjxkg nfq lol mys bbqq I lxcz"
+    hackedMessage = hackVigenere(ciphertext)
```

Comment:
```suggestion
    hacked_message = hack_vigenere(ciphertext)
```
Pythonic naming please as discussed in CONTRIBUTING.md.  Please repeat for all variable and function names.